### DISCUSSION TITLE
Treatment of source term in the flow equation in ContactMechanicsBiot model

### DISCUSSION THREAD

**Discussion by jhabriel** [pmgbergen/porepy#674] [Q&A] [answered]
Hi, 

I've noticed that in the treatment of the flow source term, a scaling with time-step is tacitly assumed,  see:
https://github.com/pmgbergen/porepy/blob/f68781ee88530b44440465f4d083243a326e08a7/src/porepy/models/contact_mechanics_biot_model.py#L835

Is this the intended treatment? Any particular reason why the scaling is not included explicitly, i.e.:

`- ad.time_step * flow_source`

I think this would be more consistent with the nature of the governing equations.

Moreover, it is not consistent with the implementation of SlightlyIncompressibleFlow, see:

https://github.com/pmgbergen/porepy/blob/f68781ee88530b44440465f4d083243a326e08a7/src/porepy/models/slightly_compressible_flow_model.py#L84

**Comment by IvarStefansson** [comment, marked as answer]
I think the answer has to do with the Biot class and the assembler version. My suggestion is that you make an issue on this and that we resolve it once we remove the assembler version of Models.
